suggestions/cloud_services.json

```diff
@@ -1 +1,30 @@
-[]
\ No newline at end of file
+[
+    {
+        "id": "com.owncloud.android",
+        "label": "ownCloud",
+        "source": "fg",
+        "reason": "Add an ownCloud server now and get... Your Cloud, Your Data, Your Way!\nhttps://owncloud.org/",
+        "repo": "https://github.com/owncloud/android"
+    },
+    {
+        "id": "com.infomaniak.drive",
+        "label": "Infomaniak kDrive",
+        "source": "fg",
+        "reason": "The swiss cloud service that doesn’t analyse or resell stored data.\nhttps://www.infomaniak.com/\nContains crash reports telemetry bits",
+        "repo": "https://github.com/Infomaniak/android-kDrive"
+    },
+    {
+        "id": "com.nextcloud.client",
+        "label": "Nextcloud",
+        "source": "fg",
+        "reason": "A safe home for all your data. Access & share your files, calendars, contacts, mail & more from any device, on your terms.\nhttps://nextcloud.com/",
+        "repo": "https://github.com/nextcloud/android"
+    },
+    {
+        "id": "io.filen.app",
+        "label": "Filen Drive",
+        "source": "g",
+        "reason": "Zero knowledge end-to-end encrypted cloud storage, redefined.\nhttps://filen.io/",
+        "repo": "https://github.com/FilenCloudDienste/filen-mobile"
+    }
```

Comment:
A work-in-progress app that does not offer any good features expected from a cloud service app on Android.

Reply:
**Filen has :**
- Automatic photo backup
- Integration in the share menu to upload files
- Files and folder sharing (like any other cloud provider)
- Is fully open-sourced and e2e encrypted on the client side

**However...**
- It has been found to be slow on startup
- It hasn't an "Android-like" user interface (it's more an iOS-like one)
- It hasn't document collaboration features, but it's planned
- It lacks integration in the SAF, but it's planned

Filen is effectively too young to belong to this list, but let's wait and track its progress, because it hasn't any anti-features the others apps have (except Nextcloud).

Reply:
> Filen is effectively too young to belong to this list, but let's wait and track its progress, because it hasn't any anti-features the others apps have (except Nextcloud).

I agree. You can open an issue for it. But it shouldn't be listed here.

Reply:
> I agree. You can open an issue for it. But it shouldn't be listed here.

Created: #33